se3cnn/SO3.py

```diff
@@ -478,32 +476,43 @@ def spherical_harmonics_coeff_to_sphere(coeff, alpha, beta):
     return s
 
 
-
 ################################################################################
 # Linear algebra
 ################################################################################
 
-def get_matrix_kernel(A, eps=1e-10):
-    """
-    Compute an orthonormal basis of the kernel (x_1, x_2, ...)
-    A x_i = 0
-    scalar_product(x_i, x_j) = delta_ij
-
-    :param A: matrix
-    :return: matrix where each row is a basis vector of the kernel of A
-    """
-    _u, s, v = torch.svd(A)
```

Comment:
You should be able to improve this generic function for any null space dimension

Reply:
I am a bit confused, can you expand on this one?

Reply:
I am wondering if your use of `eigh` is also suitable for null spaces of any dimensions

Reply:
it works for any, though you will receive not all if asked number of vectors is smaller than null space

Reply:
That was my concern. Then I guess we get rid of `get_matrix_kernel` since there is now 2 reasons to not keep it.